README.md

```diff
@@ -24,6 +24,63 @@ var sensor = instana.NewSensor("my-service")
 
 A full example can be found under the examples folder in [example/webserver/instana/http.go](./example/webserver/instana/http.go).
 
+### Log Output
+
+The Go sensor uses a leveled logger to log internal errors and diagnostic information. The default `logger.Logger` uses `log.Logger`
+configured with `log.Lstdflags` as a backend and writes messages to `os.Stderr`. By default this logger only prints out the `ERROR` level
```

Comment:
@andrewslotin we usually collect `WARN` too

Reply:
@mmanciop, good to know! Will keep it in mind when we start working on log capturing 😃 

These changes, however, are irrelevant and only address the way the Go sensor writes internal diagnostic messages. At the moment it uses an internal logger that outputs to stderr in plain text. Once this PR is merged it'll be possible to configure this behavior or even use an external logger.